resources/migrations/000_migrations.yaml

```diff
@@ -14511,6 +14511,151 @@ databaseChangeLog:
+                    referencedColumnNames: id
+                    foreignKeyName: fk_model_index_model_id
+                    deleteCascade: true
+              - column:
+                  name: pk_ref
+                  type: ${text.type}
+                  remarks: 'Serialized JSON of the primary key field ref.'
+                  constraints:
+                    nullable: false
+              - column:
+                  name: value_ref
+                  type: ${text.type}
+                  remarks: 'Serialized JSON of the label field ref.'
+                  constraints:
+                    nullable: false
+              - column:
+                  name: generation
+                  type: int
+                  remarks: 'The current generation of indexed values. Values from previous generations can be deleted.'
+                  constraints:
+                    nullable: false
+              - column:
+                  name: schedule
+                  type: ${text.type}
+                  remarks: 'The cron schedule for when value syncing should happen.'
+                  constraints:
+                    nullable: false
+              - column:
+                  name: state
+                  type: ${text.type}
+                  remarks: 'The status of the index: initializing, indexed, error, overflow.'
+                  constraints:
+                    nullable: false
+
+              - column:
+                  name: state_change_at
+                  type: ${timestamp_type}
+                  remarks: 'When the status changed'
+                  constraints:
+                    nullable: true
+              - column:
+                  name: error
+                  type: ${text.type}
+                  remarks: 'The error message if the status is error.'
+                  constraints:
+                    nullable: true
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  remarks: 'The timestamp of when these changes were made.'
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: ID of the user who created the event
+                  name: creator_id
+                  type: int
+                  constraints:
+                    nullable: false
+                    references: core_user(id)
+                    foreignKeyName: fk_model_index_creator_id
+                    deleteCascade: true
+      rollback:
+        - dropTable:
+            tableName: model_index
+
+  - changeSet:
+      id: v47.00-019
+      author: dpsutton
+      comment: Indexed Entities values table
+      changes:
+        - createTable:
+            tableName: model_index_value
+            remarks: 'Used to keep track of the values indexed in a model'
+            columns:
+              - column:
+                  name: model_index_id
```

Comment:
we need to create an index for this column

Reply:
good call out. I'm gonna added indexes to `model_index.model_id` and `model_index_value.model_index_id`